Automattic/newspack-theme
# feat(author-archive): replace twitter icon with x

### All Submissions:






### Changes proposed in this Pull Request:



Updates the Twitter bird to X's X for author archives.

### How to test the changes in this Pull Request:

1. Edit an author, with `wordpress-seo` active and at the latest version, and makes sure "X username (without @)" is filled
2. Visit the author archive, observe the X logo instead of the Twitter bird logo

### Other information:




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206366882361981
